etn-ccis/blui-react-native-workflows
# Publish Package 5.2.0

Publish Package 5.2.0


#### Changes proposed in this Pull Request:

- This PR contains changes for 6.0.0-alpha.1 (Published By the Fiji team)
- Package 5.2.0 (Fixes for Fiji team)
-



#### Screenshots / Screen Recording (if applicable)

-



#### To Test:

- **For Version 6.0.0-alpha.1**

-   Ability to hide the CreatePassword screen.
-   Ability to hide the login form in the Login screen.

- For Version 5.2.0

- Disable Next button when valid prop of Account Details form is false ([#276](https://github.com/etn-ccis/blui-react-native-workflows/issues/276)).
-   Verification code prefilled value issue on Verify Email screen on resend, next and back button pressed ([#280](https://github.com/etn-ccis/blui-react-native-workflows/issues/280)).

-   Incorrect navigation on dismissal of error occurred in Account Details screen ([#275](https://github.com/etn-ccis/blui-react-native-workflows/issues/275)).



#### Any specific feedback you are looking for?

- In **5.2.0 version** changes from **6.0.0-alpha.1** will be available . @joebochill is it fine?
